client/src/components/Brand.tsx

```diff
@@ -1,42 +1,67 @@
 import { Link } from 'react-router-dom';
+import { useContext } from 'react';
+import { closeContext } from './NavDrawerCloseContext';
 type BrandProps = {
   isOpen: boolean;
+  handleClosing: () => void;
 };
 
-export function Brand({ isOpen }: BrandProps) {
+export function Brand({ isOpen, handleClosing }: BrandProps) {
+  const { closeNavDrawer } = useContext(closeContext);
+
   return (
     <>
       {isOpen && (
-        <ul className="ml-3 cursor-pointer">
+        <ul className="ml-3 cursor-pointer" onClick={closeNavDrawer}>
```

Comment:
Since you wrap the entire return statement with `isOpen`, it would be cleaner to "fail fast". That is, check this and return, then do your happy path afterwards. Something like this:
```
if (!isOpen) return null;
return (<ul .........);
```